data/console/test_pysample_package.py

```diff
@@ -0,0 +1,8 @@
+import pysample_package
+
+print(pysample_package.__version__)
+#print(pysample_package.__author__)
+
+from pysample_package import mymodule
+mymodule.greet("Ram")
+mymodule.print_hello()
```

Comment:
you can use the package from: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17675 if you rebase on top of her changes, so you can leverage on @DeepthiYV's module